xt-bundle-aos.cfg

```diff
@@ -0,0 +1,53 @@
+[path]
+# this shall be fast drive for doing most of the build on
+workspace_base_dir = /data/storage/work/xen_prod/build
+
+[bundle_config]
+top_template = bundle_templates/top_template.json
+boardModel = hw1
+formatVersion = 1
+components = dom0 domd domf
+ostree_path = /data/.aos_ostree
+bundle_deploy_dir = /data/bundles/
+
+[dom0]
+enabled = 1
+item_template = bundle_templates/item_template.json
+version = 1
+description = some description
+requiredVersion = 1
+minimalVersion = 1
+partition_no = 1
+dom_os_name = dom0
```

Comment:
Why it is needed for bundle generation? It should be `component_id` here I guess.

Reply:
As long as Domain name may be different from the name in deploy dir - I had to add this parameter. This was mostly needed for DomF, because DomF domain is called domu in system.